src/PathDrawing/SdkModel/WayPointsFactory.cpp

```diff
@@ -0,0 +1,20 @@
+// Copyright eeGeo Ltd (2012-2015), All Rights Reserved
+
+#include "WayPointsFactory.h"
+#include "WayPoint.h"
+namespace ExampleApp
+{
+    namespace PathDrawing
+    {
+        namespace SdkModel
+        {
+            ExampleApp::PathDrawing::WayPoint* WayPointsFactory::CreateWayPoint(const int wpId
+                                                                                , const ExampleApp::PathDrawing::WayPointType::Values m_type
+                                                                                , Eegeo::Space::LatLong& m_location
+                                                                                , const std::string& m_title)
+            {
+                return Eegeo_NEW(ExampleApp::PathDrawing::WayPoint)(wpId, m_type, m_location, m_title);
```

Comment:
Does this type need a factory? The CreateWayPoint() signature is identical to the constructor, and the parameter names for this method are not the usual format (i.e. they have class member style 'm_location' instead of just 'location').  If this type is not expected to gain more dependencies or be allocated from object pools, can probably just remove this and use a constructor?